### DISCUSSION TITLE
Progress reset itself

### DISCUSSION THREAD

**Discussion by blatisgense** [codewars/codewars.com#3194] [Q&A] [answered]
Hello, I've got a serious problem. My account score erased when I started to train TypeScript after JavaScript. How can I return my 1180 JavaScript points to display within my profile? I worked hard for that score and don't wanna lose it. My username is Blatisgense

**Comment by Blind4Basics** [comment]
Hello,

Who is `Js.Nutshell`?

**Reply by Blind4Basics** [reply]
`Js.nutshell` is a copycat. I'm actually a bit surprized "your lil brother" didn't already tell you his account is in the same state as yours...

You also used solutions that are not your own and your account got locked and progress were reset because of that. So this has nothing to do with you starting training in TS.

**Reply by Blind4Basics** [reply]
Please note I'm not saying _all_ your solutions are copies. But don't dare argue that you authored all of them, it's a blatant lie.
You should delete your account and start fresh. And this time, _actually_ author all your solutions.

edit: same for your brother, ofc.

**Reply by Blind4Basics** [reply, marked as answer]
* if you delete the current one, you can use the same username for the new account.
* if you directly copy your old solutions (those you actually wrote, I mean), you _might_ get into troubles again, yes. If you do that in the next bunch of hours, I can double check if you want. But note that if any of the previously copied (as in "not your own") solutions are found, the new account will also get locked.

**Reply by Blind4Basics** [reply]
I invalidated one so far. Next time, I lock the account.

In case it's not clear enough: rewritting existing solutions won't save you from being considered a copycat...
